port_drayage_plugin/src/port_drayage_plugin.cpp

```diff
@@ -29,10 +29,13 @@ namespace port_drayage_plugin
 
         double speed_epsilon = _pnh->param("stop_speed_epsilon", 1.0);
         declaration = _pnh->param("declaration", 1.0);
-        std::string cmv_id;
-        _pnh->param<std::string>("cmv_id", cmv_id, "");
         std::string cargo_id;
-        _pnh->param<std::string>("cargo_id", cargo_id, "");
+        _pnh->param<std::string>("cargo_id", cargo_id, ""); 
+
+        // Read in 'cmv_id' parameter as a string, then convert to an unsigned long before initializing the PortDrayageWorker object
+        std::string cmv_id_string;
+        _pnh->param<std::string>("cmv_id", cmv_id_string, "123");
```

Comment:
When possible let's make these kind of default values very obviously wrong "UNDEFINED" or something similar, easier to track if they do show up in some data stream somewhere somehow.

Reply:
Great point. I've updated cargo_id to "UNDEFINED-CARGO-ID".  I was unable to change cmv_id to a more descriptive default value since it is converted to an unsigned long.